TARS/src/common/ServoControl.h

```diff
@@ -4,6 +4,11 @@
 
 #include <PWMServo.h>
 
+#define A 9.8
+#define B 2.05
+#define C -154
+#define D 4.1
```

Comment:
If we put this in the header then any use of the symbols A, B, C, or D in the entirety of flight code will be replaced by random numbers. This currently isn't an issue, but we should move this to ServoControl.cpp just to avoid this issue in the future.